charon/src/bin/charon-driver/translate/translate_crate_to_ullbc.rs

```diff
@@ -358,6 +359,28 @@ impl<'tcx, 'ctx> TranslateCtx<'tcx, 'ctx> {
         }
         Ok(self.translated.get_item(id).unwrap())
     }
+
+    /// Return a map from source file name to file content.
+    fn read_source_files(&mut self) -> HashMap<FileName, Arc<String>> {
+        // Retrieve the source map
+        let source_map = self.tcx.sess.source_map();
+
+        // Read all the files
```

Comment:
Could you mention that this includes all the files of all the dependent crates too and hence we should be careful?

Reply:
Yes, good idea